index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>LAB-2</title>
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/nav-panel.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+      integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr"
+      crossorigin="anonymous"
+    />
+    <meta charset="UTF-8" />
+  </head>
+  <body>
+    <div class="menu-btn" id="main-menu" onclick="clickMenu()">
+      <i class="fas fa-bars"></i>
+    </div>
+    <div class="sidenav" id="side-nav">
+      <div class="nav-btns">
+          <a href="">PSD WEBSITES</a>
+          <a href="">UI KITS</a>
+          <a href="">PREMIUM PSD FILES</a>
+          <a href="">ABOUT</a>
+          <a href="">BLOG</a>
+        </div>
+    </div>
+    <div class="topnav">
+      <div class="nav-btns">
+        <a href="">PSD WEBSITES</a>
+        <a href="">UI KITS</a>
+        <a href="">PREMIUM PSD FILES</a>
+        <a href="">ABOUT</a>
+        <a href="">BLOG</a>
+      </div>
+    </div>
+    </div>
+    <div class="banner">
+      <div>Awesome Free PSD<br />templates & Resources</div>
+      <div class="link-text">
+        www.BlazRobar.com
+      </div>
+    </div>
+    <div class="text-box">
+      <div class="text-box-text">
+        I'm like the Robin Hood of PSD files.
+      </div>
+      <div class="arrow-box">
+        <i class="fas fa-angle-right"></i>
+      </div>
+    </div>
+    <div class="media">
+      <img src="img/1.png" class="item" />
+      <img src="img/2.png" class="item" />
+      <img src="img/3.png" class="item" />
+    </div>
+    <div class="info">
+      <h1>BlazRobar.com</h1>
+      <div class="content">
```

Comment:
I'd recommend to use [semantic tags](https://www.w3schools.com/html/html5_semantic_elements.asp) 

[Why use semantic HTML?](https://www.lifewire.com/why-use-semantic-html-3468271)